doc/news/_preparation_next_release.md

```diff
@@ -50,11 +50,10 @@ The following section lists news about the [modules](https://www.libelektra.org/
 
 - We removed the plugin in favor of [Yan LR](../../src/plugins/yanlr/README.md). _(René Schwaiger)_
 
-### <<Plugin2>>
+### Type
 
-- <<TODO>>
-- <<TODO>>
-- <<TODO>>
+- We added an option to disable the restoring of boolean values. This useful for storage formats like YAML that have
```

Comment:
You should either:

- mention here that the option was renamed
- not rename it, I think boolean/restore is still perfectly suitable even if it gets the additional option "none".

Reply:
The option was always called `boolean/restoreas`. There was just a typo in the README.

The relevant line in the code remained unchanged:

https://github.com/ElektraInitiative/libelektra/blob/9420c349af265cc427a98fad30466fc0d696c619/src/plugins/type/type.c#L198

Reply:
Ahh, ok. Then it is okay as-is. A bit more info in the release notes would not harm.

Nevertheless, I would prefer boolean/restore.

Reply:
I chose the name `restoreas`, because we always restore (except with the new `none` option). The config key just changes _how_ we restore.

Since it would be a breaking change to rename it now and there is no actual need for it, we will leave it as is.

Reply:
The exception is essential: now the option decides whether to restore at all, so "boolean/restore" would fit better.

> Since it would be a breaking change to rename it now and there is no actual need for it, we will leave it as is.

Within 0.9 we can do such changes. A simple sed should do, doesn't it?